test/morse_translator_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'pry'
+
+
+require 'minitest/autorun'
+require 'minitest/pride'
+
+require './lib/morse_translator'
+
+class MorseTranslatorTest < Minitest::Test
```

Comment:
You created 3 helper methods in your translator - I would have expected to see tests for each of those methods here!